src/components/performance/export.cjsx

```diff
@@ -41,8 +45,13 @@ PerformanceExport = React.createClass
         downloadUrl: lastExport.url
         lastExported: lastExport.created_at
 
+      @triggerDownload(lastExport.url) if @state.triggerDownload
       @setState(exportState)
 
+  triggerDownload: (downloadUrl) ->
+    window.location = downloadUrl
```

Comment:
Should the download replace the current browser tab (and hope that the file is downloaded rather than something like opened in the browser)?